app/src/main/java/com/example/sharingang/users/User.kt

```diff
@@ -29,6 +29,9 @@ data class User(
     /** URLs of the images */
     val profilePicture: String? = null,
 
+    var rating: Int = 0,
+    var numberOfRatings: Int = 0,
+
     var wishlist: List<String> = ArrayList(),
```

Comment:
Would be better to make it immutable too

Reply:
and done